lib/internal/crypto/cfrg.js

```diff
@@ -53,7 +53,14 @@ function verifyAcceptableCfrgKeyUse(name, type, usages) {
     case 'X25519':
       // Fall through
     case 'X448':
-      checkSet = ['deriveKey', 'deriveBits'];
+      switch (type) {
+        case 'private':
+          checkSet = ['deriveKey', 'deriveBits'];
+          break;
+        case 'public':
+          checkSet = [];
+          break;
+      }
```

Comment:
Is a switch relevant here (rather than say a ternary)? I think we should add an `assert(false)` in a default case if we keep the switch.
```suggestion
      assert(type === 'private' || type === 'public');
      checkSet = type === 'private' ? ['deriveKey', 'deriveBits'] : [];
```

Reply:
I'd like to stick to the same style of code present in cfrg, ec, and rsa right now. I'd be happy to refactor this in a follow up separate PR.